### DISCUSSION TITLE
IgG perturbations

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#1120] [Questions & Problems]
Dear community, 
Hope you are doing well! I was wondering what underlying processes are already put in place for IgG in the therapeutic protein model. More specifically, do the current models already allow to predict the perturbation of IgG concentrations in plasma after dosing it (IgG therapy) or depleting it (e.g. to reduce antibody response to certain therapies)? When I try to export a random model from PKsim to Mobi and look at the predicted IgG concentrations (which is called EndoLigand, correct?) they are shown as a stable line, but when I e.g. look at the Idarucizumab, the IgG pool (or at least the EndoLigand pool) seems to deplete. Any insights are very welcome!

Thanks,
Robin

**Comment by Christoph27** [comment]
Hi Robin,

There is a submodel for endogenous IgG (indeed called LigandEndo in the implementation) and FcRn lumping the whole organism within a single PK-Sim organ as described in https://doi.org/10.1007/s10928-017-9559-4. 
So - given the model of an “antibody A” - if you add an application for LigandEndo into the plasma compartment of the “EndogenousIgG organ” in MoBi at a certain timepoint, you can see the expected behavior (increase of endogenous IgG concentration at that time point and slow return to the steady state of the endogenous IgG and increase in clearance for the “antibody A” starting at the timepoint of the application of the endogenous IgG. I have however never tested how well it works quantitatively with the current parameterization. 
(What will not work, is to change the steady state start concentration of endogenous IgG (if it is e.g. constantly depleted) because the parameters for the steady state start concentrations for both endogenous IgG and FcRn refer to the free concentrations. One could then directly adapt the steady state start concentrations of free FcRn.)

I’m wondering why you mention Idarucizumab in this context. Isn’t it a Fab fragment, i.e. it should not compete with the endogenous IgG for FcRn?

**Comment by RobinM92** [comment]
Hi Christoph,

Thank you for your answer! Good to know that I at least got the IgG name 
right :)
I don't completely follow with the second part; why does the fact that 
IgG and FcRn are free concentrations limits one from including an 
elimination process of IgG (e.g. depletion in a disease state)?

The Idarucizumab model was just one of the ready to play models I found 
in the repository which I used to explore the behaviour of IgG, I had no 
interest in the compound itself (although it is a neat model :) )

Kind regards,
Robin

Dr. ir. Robin Michelet
Senior scientist

Freie Universitaet Berlin
Institute of Pharmacy
Dept. of Clinical Pharmacy & Biochemistry
Kelchstr. 31
12169 Berlin
Germany
Phone:  + 49 30 838 50659
Fax:      + 49 30 838 4 50656
***@***.***
www.clinical-pharmacy.eu
https://fair-flagellin.eu/

**Comment by Christoph27** [comment]
Hi Robin,

it is completely fine to add a depletion process for endogenous IgG. I was just wondering if you might have tried to just change the steady state start concentration of free endogenous IgG and were thinking why nothing is happening to the clearance of the drug antibody (The reason – if not clear from above – is that for the clearance the available free FcRn is relevant which is not changing if not adjusted. In contrast to the situation when changing the total IgG concentration which would change the free FcRn concentration if total FcRn concentration is kept constant). 

All the best,
Christoph

**Reply by RaphBil** [reply]
Hi @Christoph27,
Following up on your reply. Is that mean the free FcRn levels are not impacted by administration of a mAb with very high affinity?
In other words, potential interaction at the FcRn level between two mAbs (with high FcRn binding at pH6.0) co-administered is not going to be reflected using the default model? The FcRn free levels will be sustained at initial steady-state levels anyway?

Thank you.
Best,
Raphael

**Reply by StephanSchaller** [reply]
Dear @RaphBil , sure, FcRn free levels are changing, can even be saturated.
What @Christoph27 hris is writing is related to changing initial conditions of endogenous IgG.

**Reply by Christoph27** [reply]
As Staphan wrote, I was just referring to the initial conditions (one has to keep in mind that the corresponding parameter is free FcRn, not total FcRn). Free FcRn levels are impacted by administered Ab.
